README.md

```diff
@@ -126,22 +132,22 @@ Current release: `1.0.0-beta7`
 
 ### v1.0 TODO:
 
-- [x] Update docs for Button component
-- [x] Update docs for Input component
-- [x] Update docs for Search component
-- [x] Write detailed release notes with new features & breaking changes
-- [x] Update react, react-native & other dependencies
-- [x] Update docs for ListItem component
-- [x] Write docs for Themes support
-- [ ] Write docs for migration from v0.19.1 to 1.0.0 -
-      **[\$200](https://opencollective.com/react-native-elements#budget)**
-- [ ] Create a compelling demo/example app for RNE + RNW (Checkout
+* [x] Update docs for Button component
+* [x] Update docs for Input component
+* [x] Update docs for Search component
+* [x] Write detailed release notes with new features & breaking changes
+* [x] Update react, react-native & other dependencies
+* [x] Update docs for ListItem component
+* [x] Write docs for Themes support
+* [ ] Write docs for migration from v0.19.1 to 1.0.0 -
+      **[$200](https://opencollective.com/react-native-elements#budget)**
+* [x] Create a compelling demo/example app for RNE + RNW (Checkout
       [rn-web](https://github.com/react-native-training/react-native-elements/tree/rn-web)
```

Comment:
feel free to delete this part. Just "Create a compelling demo/example app for RNE + RNW" should be enough

Reply:
@Monte9 done 👌